[4.4] docs(pivotgrid): add title parameter (#1541)
telerik/blazor-docs: components/pivotgrid/overview.md

| `Aggregate` | `PivotGridAggregateType` enum <br /> (`Sum`) | The nature of the calculated aggregate values. Applies to `PivotGridMeasure` only. |
| `Format` | `string` | The [display format](https://learn.microsoft.com/en-us/dotnet/standard/base-types/standard-numeric-format-strings) of the calculated aggregate values, for example `"{0:C2}"`. Applies to `PivotGridMeasure` only. |
| `HeaderClass` | `string` | Adds a custom CSS class to the row/column/measure header. Use it to apply custom styles or [override the default PivotGrid styles]({%slug themes-override%}). |

> | `Name` | `string` | The field name of the respective row/column/measure. |
> | `Title` | `string` | The text that is rendered in the row/column/measure header. |

Reviewer: Are you sure there are titles for measures? Or are we talking about labels for the Configurator? In the latter case, we shouldn't use "header", as it may be misleading.

```suggestion
| `Name` | `string` | The field name of the respective row, column or measure. |
| `Title` | `string` | The label to be displayed in the Configurator for the respective row, column or measure. |
```

Author: I confirm that we are talking about the labels for the configurator so the suggestion is completely valid.
